assets/css/abstracts/_variables.scss

```diff
@@ -23,3 +23,12 @@ $blue-alt-light:#8caabb;
 $green:#b0d0be;
 $green-light: #cee6c5;
 $green-pale: #f7fff4;
+
+// UIO
+$yellow: #ff0;
+$white: #fff;
+$grey: #bdbdbb;
+
+// title
+$title-size: 4em;
+$title-weight: semi-bold;
```

Comment:
I have removed the title variable in another PR

Reply:
Once that PR is merged I will update this.